web/scm_helpers.go

```diff
@@ -30,27 +30,31 @@ var (
 )
 
 // createRepoAndTeam invokes the SCM to create a repository and team for the
-// specified namespace (typically the course name), the path of the repository
-// (typically the name of the student with a '-labs' suffix or the group name).
-// The team name is the student name or group name, whereas the user names are
-// the members of the team. For single student repositories, the user names are
-// typically just the one student's user name.
+// specified namespace (typically the course name), and teamName (name of the group).
```

Comment:
I don't recall why I wrote namespace here, but it would be helpful if this were reflected in the variable names. Maybe it should instead refer to `orgID`, e.g.: *... and team for the specified course (represented with organization ID). The SCM team name is also used as the group name and repository path. The provided user names represent the SCM group members.*

Reply:
Fixed.